prov/src/main/java/org/bouncycastle/jcajce/provider/asymmetric/GM.java

```diff
@@ -30,6 +30,12 @@ public void configure(ConfigurableProvider provider)
         {
             provider.addAlgorithm("Signature.SM3WITHSM2", PREFIX + "GMSignatureSpi$sm3WithSM2");
             provider.addAlgorithm("Alg.Alias.Signature." + GMObjectIdentifiers.sm2sign_with_sm3, "SM3WITHSM2");
+            // SHA1 with SM2
+            provider.addAlgorithm("Signature.SHA1WithSM2", PREFIX + "GMSignatureFullSpi$sha1WithSM2");
```

Comment:
Why does this say GMSignatureFullSpi instead of GMSignatureApi? (same problem 3 lines below).

Reply:
this is my develop file , oh sory for that I`ll fix it